rollout/bluegreen.go

```diff
@@ -71,18 +71,14 @@ func (c *rolloutContext) rolloutBlueGreen() error {
 	return c.syncRolloutStatusBlueGreen(previewSvc, activeSvc)
 }
 
-func (c *rolloutContext) reconcileBlueGreenStableReplicaSet(activeSvc *corev1.Service) error {
-	if _, ok := activeSvc.Spec.Selector[v1alpha1.DefaultRolloutUniqueLabelKey]; !ok {
-		return nil
-	}
-	activeRS, _ := replicasetutil.GetReplicaSetByTemplateHash(c.allRSs, activeSvc.Spec.Selector[v1alpha1.DefaultRolloutUniqueLabelKey])
-	if activeRS == nil {
-		c.log.Warn("There shouldn't be a nil active replicaset if the active Service selector is set")
+func (c *rolloutContext) reconcileBlueGreenStableReplicaSet() error {
+	if c.stableRS == nil {
+		c.log.Info("stableRS is nil on first deployment, we don't need to scale it in this case")
```

Comment:
This log message talks about `scaling` which seems unintuitive as the method is simply trying to reconcile (scaling is done as part of the reconciliation)

You can just remove the log or change it to say, _"Stable ReplicaSet doesn't exist and hence no reconciliation is required."_

Reply:
Thanks for the input, I've updated the comment to what you suggested.

Reply:
Thanks! Could you please also add a test for this scenario?

Reply:
Hmm, not sure what exactly needs testing here. Other tests which initialize a deployment are inherently testing this path, but there is nothing measurable to test in this case? Scaling the rollout during deployment wouldn't help either as the other methods during the reconciliation process would cause the replicaset to scale, meaning we cannot expect _nothing_ to happen

Any help/specifics would be awesome for this one.

Reply:
Could you add an explicit test case which asserts on whether the log line: _"Stable ReplicaSet doesn't exist and hence no reconciliation is required."_ is emitted?

Reply:
Right, I've attempted that - hopefully it looks okay.